README.md

```diff
@@ -21,26 +31,35 @@ modules
 Inside each one the format would be:
 
 ```js
-module.exports = (self, options) => {
-  return {
-    ... a3 fields here
-  };
+module.exports = {
+  ... a3 fields here
 };
 ```
 
 This makes it easy to pull into custom-module/index.js:
 
 ```js
 module.exports = {
-  fields(self, options) => require('./schema.js')(self, options)
+  extend: '@apostrophecms/piece-type', // or whatever type needed as '@apostrophecms/piece-page-type' for example
+  fields: require('./schema.js')
 };
 ```
 
+Another option is to output an entire module instead of the `fields` object by passing the option `--module`.
```

Comment:
I think this suggests you're going to get all of the code from the original A2 module, which you aren't, so it is dangerous to offer it. I don't see why we would include this in a schema exporter, it's outside the scope.

Reply:
Done